fastlane/lib/fastlane/plugins/plugin_manager.rb

```diff
@@ -343,9 +343,12 @@ def store_plugin_reference(gem_name)
       references = Fastlane.const_get(module_name).all_classes.collect do |path|
         next unless File.dirname(path).end_with?("/actions") # we only want to match actions
 
-        File.basename(path).gsub("_action", "").gsub(".rb", "").to_sym # the _action is optional
+        File.readlines(path)
+          .map { |line| line.sub!(/class (.*) < Action/, '\1') }
```

Comment:
Ugh, not sure if parsing the actual file content is a good idea, it seems not to be a stable approach 😢